src/influxdb_ioxd/rpc/flight.rs

```diff
@@ -70,7 +70,19 @@ impl From<Error> for tonic::Status {
     /// Converts a result from the business logic into the appropriate tonic
     /// status
     fn from(err: Error) -> Self {
-        error!("Error handling Flight gRPC request: {}", err);
+        // An explicit match on the Error enum will ensure appropriate
+        // logging is handled for any new error variants.
+        let msg = "Error handling Flight gRPC request";
+        match err {
+            Error::DatabaseNotFound { .. }
+            | Error::InvalidTicket { .. }
+            | Error::InvalidQuery { .. }
+            | Error::InvalidDatabaseName { .. } => debug!(?err, msg),
```

Comment:
```suggestion
            | Error::InvalidDatabaseName { .. } => info!(?err, msg),
```
what do you think about logging this at `info` for a while more so they are captured in greylog? Especially as I think we use the `DatabaseNameNotFound` to figure out if we need to configure more databases

Reply:
I think that's ok. I will update. In general though I think it would be good to just be able to run IOx at "debug" level logging if that's the level we need whilst we ramp up.